ui/src/app/components/app-router.tsx

```diff
@@ -124,12 +124,12 @@ export const RouteLink = ({
   children,
 }): React.ReactElement => {
   const linkStyles = { textDecoration: 'none', color: 'unset' };
-  return !disabled ? (
```

Comment:
To be sure, it has reverted the logic i.e use disabled then do this rather than enabled and do this

And add cursor: 'not-allowed' style in case its enabled?

Reply:
fixed

Reply:
Looks good to me